packages/compas-convert/src/passes/convert-test-files.ts

```diff
@@ -128,7 +132,7 @@ function handleNestedTest(expression: CallExpression) {
 				const expected = args[1]?.getText() ?? "true";
 				const message = args[2]?.getText() ?? "";
 				it.expression = it.expression.replaceWithText(
-					`expect(${actual}${message ? `, ${message}` : ""}).toStrictEqual(${expected})`,
+					`expect(${actual}${message ? `, ${message}` : ""}).toEqual(${expected})`,
```

Comment:
Strict equal might be too strict? This is less strict on `undefined` vs missing keys for example.